docs/docs/concepts/repos.md

```diff
@@ -0,0 +1,80 @@
+# Repos
+
+Running [`dstack apply`](../reference/cli/index.md#dstack-apply) in a folder uploads its contents and mounts them to the container’s `/workflow` directory. 
+This makes files from the folder accessible within the configuration.
+
+## Initialize a repo
+
+Before using [`dstack apply`](../reference/cli/index.md#dstack-apply), initialize the folder as a repo by running [`dstack init`](../reference/cli/index.md#dstack-init) once. 
+The folder can be either a local directory or a remote Git repo.
+
+### Git credentials
+
+If the folder is a remote Git repo, [`dstack init`](../reference/cli/index.md#dstack-init) grants the `dstack` server access by uploading the current user's default
```

Comment:
> If the folder is a remote Git repo

Sounds clumsy. I'd avoid using "folder" altogether and used "directory" for consistency.